deepmd/utils/argcheck.py

```diff
@@ -1176,6 +1176,37 @@ def fitting_dipole():
     ]
 
 
+@fitting_args_plugin.register("property")
+def fitting_property():
+    doc_neuron = "The number of neurons in each hidden layers of the fitting net. When two hidden layers are of the same size, a skip connection is built."
+    doc_activation_function = f'The activation function in the fitting net. Supported activation functions are {list_to_doc(ACTIVATION_FN_DICT.keys())} Note that "gelu" denotes the custom operator version, and "gelu_tf" denotes the TF standard version. If you set "None" or "none" here, no activation function will be used.'
+    doc_resnet_dt = 'Whether to use a "Timestep" in the skip connection'
+    doc_precision = f"The precision of the fitting net parameters, supported options are {list_to_doc(PRECISION_DICT.keys())} Default follows the interface precision."
+    doc_seed = "Random seed for parameter initialization of the fitting net"
+    doc_task_num = "The dimension of outputs of fitting net"
```

Comment:
I mean the variable name should be self-explanary like "task_dim" instead of "task_num"